### ISSUE TITLE
Questions about training

### ISSUE THREAD

**Issue by 7LuoXi** [uzh-rpg/rpg_e2depth#3]
Hi, thanks for your work!
Could you please release your training code or the implementation of loss function?

**Comment by urancon** [comment]
Hi, thanks for your paper and the repo !

Like @7LuoXi , I am interested in your training code and your implementation of the loss function too.
Also, the version of the model currently on the repo does not seem to output intermediate depth maps (used for the multiscale loss); how did you retrieve them from the network in practice ? Did you use specific layers ?

Thank you in advance for your answer !

**Comment by gavinatthu** [comment]
> Hi, thanks for your paper and the repo !
> 
> Like @7LuoXi , I am interested in your training code and your implementation of the loss function too.
> Also, the version of the model currently on the repo does not seem to output intermediate depth maps (used for the multiscale loss); how did you retrieve them from the network in practice ? Did you use specific layers ?
> 
> Thank you in advance for your answer !

I think their new released project ramnet may help you[https://github.com/uzh-rpg/rpg_ramnet/tree/master/RAM_Net](url)

**Comment by urancon** [comment]
Oh, I'll check that, thanks so much for telling !
